lib/livebook/runtime/erl_dist/runtime_server.ex

```diff
@@ -446,6 +450,11 @@ defmodule Livebook.Runtime.ErlDist.RuntimeServer do
     {:noreply, state}
   end
 
+  def handle_cast({:add_secrets, secrets}, state) do
```

Comment:
It's better to use more granular operations, in this case for adding a single secret. This way if multiple users add a different secret at the same time, both changes are reflected. I'd call it `put_secret` as it adds or updates. Then, if we add secret removal, we will also have `delete_secret`.